app/code/Magento/Sales/Test/Mftf/ActionGroup/AssertOrderAddressInformationActionGroup.xml

```diff
@@ -7,14 +7,16 @@
 -->
 
 <actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-        xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
     <actionGroup name="AssertOrderAddressInformationActionGroup" extends="VerifyBasicOrderInformationActionGroup">
         <remove keyForRemoval="seeCustomerName"/>
         <remove keyForRemoval="seeCustomerEmail"/>
         <remove keyForRemoval="seeCustomerGroup"/>
         <remove keyForRemoval="seeBillingAddressCountry"/>
         <remove keyForRemoval="seeShippingAddressCountry"/>
         <see selector="{{AdminOrderAddressInformationSection.billingAddress}}" userInput="{{billingAddress.country}}" stepKey="seeBillingCountry" after="seeBillingAddressCity"/>
+        <see selector="{{AdminOrderAddressInformationSection.billingAddress}}" userInput="{{billingAddress.state}}" stepKey="seeBillingAddressState" after="seeBillingCountry"/>
         <see selector="{{AdminOrderAddressInformationSection.shippingAddress}}" userInput="{{shippingAddress.country}}" stepKey="seeAddressCountry" after="seeShippingAddressCity"/>
+        <see selector="{{AdminOrderAddressInformationSection.shippingAddress}}" userInput="{{shippingAddress.state}}" stepKey="seeShippingAddressState" after="seeAddressCountry"/>
```

Comment:
I'm just wondering, if this is something specifically that we need in our case, because theoretically it's BIC to adjust the existing ActionGroups, as there may be some 3rd party that can rely on this Action Group 🤔.

```suggestion
        <see selector="{{AdminOrderAddressInformationSection.billingAddress}}" userInput="{{billingAddress.state}}" stepKey="seeBillingAddressState" after="seeBillingCountry"/>
        <see selector="{{AdminOrderAddressInformationSection.shippingAddress}}" userInput="{{shippingAddress.country}}" stepKey="seeAddressCountry" after="seeShippingAddressCity"/>
        <see selector="{{AdminOrderAddressInformationSection.shippingAddress}}" userInput="{{shippingAddress.state}}" stepKey="seeShippingAddressState" after="seeAddressCountry"/>
```

Reply:
Hi @eduard13 
Agree, I'll rework this part

Reply:
@eduard13 what do you think about adding new action group that extends `AssertOrderAddressInformationActionGroup`?

Reply:
I was also thinking about that, and it might be a good idea in our case 👍